### DISCUSSION TITLE
What are "views"

### DISCUSSION THREAD

**Discussion by chrismeats** [etsy/open-api#1006] [Q&A]
I am wondering if there is more information on how "views" are calculated on a listing returned in the API.

Is it ALL views the listing has received since created or for a specific timeframe?

**Comment by Gareth064** [comment]
Shouldn't be too hard to figure out. Get a listing, see what the views value is. Then check the stats of the listing on the website and see if any timeframe comes close.

**Comment by emergingdzns** [comment]
It is supposed to be all views for the listing ever up to the day before you request that information. Etsy tabulates the data nightly and updates the table. This is why you won't see the views number change during the day.

However, there is currently a bug in that the API is returning numbers inconsistent with what is shown in the Etsy UI. There's a support ticket in the system for this.
